tikv-configuration-file.md

```diff
@@ -300,6 +300,21 @@ Configuration items related to the sharing of block cache among multiple RocksDB
 + Default value: 45% of the size of total system memory
 + Unit: KB|MB|GB
 
+## storage.io-rate-limit
+
+Configuration items related to the I/O rate limiter.
+
+### `max-bytes-per-sec`
+
++ Maximum I/O bytes that this server can write to or read from disk (determined by mode) in one second. Internally it prefers throttling background operations over foreground ones. This value should be set to the disk's optimal IO bandwidth, e.g. maximum IO bandwidth specified by cloud disk vendors. When set to zero, disk IO operations are not limited.
++ Default value: `"0MB"`
+
+### `mode`
+
++ Determine which types of IO operations are counted and restrained below threshold. Only write-only mode is supported for now.
++ Optional value: `"write-only"`, `"read-only"`, `"all-io"`
```

Comment:
If  the write-only mode is supported for now, what about removing `"read-only"` and `"all-io"` for now?
```suggestion
+ Value options: `"write-only"`, `"read-only"`, `"all-io"`
```

Reply:
I'm not sure. If so this configuration item doesn't make a lot of sense to exist. Might as well remove it? What do you think?

Reply:
And BTW the expression `Optional value` is used several times before in this document. Do we need to change them as well?

Reply:
+ If it affects the user behavior, this config should be documented. If `"read-only"` and `"all-io"` are currently not supported, we should remove them and add them back when available.
+ OK. I've deleted the suggested comment. Thanks for reminding~